### DISCUSSION TITLE
Release 3.4.3

### DISCUSSION THREAD

**Discussion by KingT1123** [lagdo/tontine#58] [General]
Bonjour à vous, merci pour l'application.
J'aime beaucoup Mais je n'arrive pas à le (v3.4.3) lancer sur ma machine. 
Ça me dit "**Unable to write cache file**" 
Pouvez vous m'aider ?

**Comment by feuzeu** [comment]
Bonjour,

Je suppose que vous avez installé PHP sur un serveur Linux?
Vérifiez que le répertoire `storage` dans votre installation est accessible en écriture à l'utilisateur qui exécute PHP, et que le répertoire `storage/annotations` est bien présent, et également accessible en écriture.

**Reply by KingT1123** [reply]
Je l'ai sur windows 11. J'ai verifie et c'est bien accessible en ecriture.
Mais je ne trouve pas le dossier annotations dans storage. Je ne sais pas pourqoui
Je ne pense pas que c'esr disponible dans le projet.

**Reply by feuzeu** [reply]
Dans ce cas, il suffit de créer le dossier `storage/annotations` à la main.
Il  aurait dû être créé à l'installation par Composer.

**Reply by KingT1123** [reply]
Quelle version de php devons nous utiliser ? car j'utilise la version 8.2 et ca ne migre pas toutes les tables correctement vers la BD.

**Reply by feuzeu** [reply]
Dans ce cas, il faut poster un message dans la section [Issues](https://github.com/lagdo/tontine/issues), en fournissant le plus d'infos possible pour comprendre le problème.
- L'OS et sa version
- La version de PHP
- Le type et la version de la base de données
- L'action ou la commande qui cause le problème
- La description du problème
- le cas échéant, le message d'erreur qui s'affiche dans le fichier de logs de Laravel.

**Reply by KingT1123** [reply]
Okay i did that.
I have another problem, how can i connect myself as a member, admin or guest locally (while running the code on my computer).
